src/networking/APNSNetworkService.php

```diff
@@ -92,6 +100,7 @@ public function enqueue_request( string $url, array $headers, string $body ): vo
 		curl_setopt( $ch, CURLOPT_POSTFIELDS, $body );
 		curl_setopt( $ch, CURLOPT_VERBOSE, $this->debug );
 		curl_setopt( $ch, CURLOPT_PORT, $this->port );
+		curl_setopt( $ch, CURLOPT_TIMEOUT, $this->timeout );
```

Comment:
👍 

I double-checked that the option expected seconds, which it does. Also learned there is a millisecond counterpart: `CURLOPT_TIMEOUT_MS`. Neat.